support-frontend/assets/components/checkoutHeading/checkoutHeading.tsx

```diff
@@ -26,6 +26,14 @@ const headingContentContainer = css`
 	${textSans.large({ fontWeight: 'bold' })}
 `;
 
+const leftColImageEditions = css`
+	height: 129px;
+
+	img {
+		max-width: 100%;
+	}
+`;
+
```

Comment:
Should we do this in [`digitalSubscriptionLandingPage`](https://github.com/guardian/support-frontend/pull/5484/files#diff-f37c28087b1494529802a18f41b7ad6c1bb6605c699e81dea0556fc3c269c656) similar to how we do in the [`/contribute` checkoutScaffold](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/pages/supporter-plus-landing/twoStepPages/checkoutScaffold.tsx#L206-L214)?

The reason being:
- it decouples this styling specific to the digital subscriptions page so any other component that uses `checkoutHeading` doesn't inherit these styles
- it also means we won't have to remove the `<figure>` from [checkoutScaffold](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/pages/supporter-plus-landing/twoStepPages/checkoutScaffold.tsx#L206-L214) as we're currently rendering `<figure><figure><!-- ... --></figure></figure>` on the `/contribute` page.

Reply:
Anything that removes unnecessary markup gets the ol' 👍 from me

Reply:
Great spot @jamesgorrie , second point 2 is an important issue need to check for in future. Should of checked dependencies on `checkoutHeading` component when using local css otherwise `<figure>`  does indeed get duplicated here. Have moved it up to `digitalSubcriptionLandingPage` now as suggested. Thanks, done.